app/models/blog.rb

```diff
@@ -10,7 +10,7 @@ class Blog < ApplicationRecord
   scope :published, -> { where('secret = FALSE') }
 
   scope :search, lambda { |term|
-    where("title LIKE '%#{term}%' OR content LIKE '%#{term}%'")
+    where('title LIKE ? OR content LIKE ?', "%#{sanitize_sql(term.to_s)}%", "%#{sanitize_sql(term.to_s)}%")
```

Comment:
同じパラメータには名前を付けてあげるとDRYにできます

```suggestion
    where('title LIKE :hoge OR content LIKE :hoge', hoge: "%#{sanitize_sql(term.to_s)}%")
```

Reply:
DRYにしました。